automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -61,34 +76,52 @@ trait GPAllocUtils extends BillingFixtures with LeonardoTestUtils {
                                                       BillingProject.BillingProjectRole.User)(hermioneAuthToken)
       )
       _ <- loggerIO.info(s"Billing project claimed: ${claimedBillingProject.projectName}")
-    } yield GoogleProject(claimedBillingProject.projectName)
+      workspaceName = UUID.randomUUID().toString
```

Comment:
Nit: maybe `workspaceName <- IO(UUID.randomUUID().toString)` since it's technically a side effect